### PR TITLE
LenovoLegionLinux Stable PKGBUILDS

### PR DESCRIPTION
**Pull request by MrDuartePT**
This PR add the stable PKGBUILDS for LenovoLegionLinux

### CODE REVIEW

File: lenovolegionlinux-dkms/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+license=('GPL')
+makedepends=(
+	"lm_sensors"
+	"i2c-tools" 
+	"dmidecode"
+	"lenovolegionlinux-git"
+)
+depends=(
+	"dkms"
+    "lenovolegionlinux-git"
```
Comment (1Naim) [inline lenovolegionlinux-dkms/PKGBUILD:21]:
```suggestion
    "lenovolegionlinux"
```
Should be this right?

File: lenovolegionlinux-dkms/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# Maintainer: johnfanv2 <https://github.com/johnfanv2>
+
+_pkgname=lenovolegionlinux
+pkgname=${_pkgname}-dkms
+pkgver=0.0.18
+pkgrel=0
+pkgdesc="LenovoLegionLinux (LLL) DKMS module"
+arch=("x86_64")
+url="https://github.com/johnfanv2/LenovoLegionLinux"
+license=('GPL')
```
Comment (ptr1337) [inline lenovolegionlinux-dkms/PKGBUILD:12]: Please set the correct SPDX Identifier

File: lenovolegionlinux-dkms/PKGBUILD (hunk 1 above)
Comment (ptr1337) [inline lenovolegionlinux-dkms/PKGBUILD:21]: tabs vs spaces

File: lenovolegionlinux-dkms/lenovolegionlinux.install [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+post_remove() {
+	rmmod legion-laptop.ko #unload old module
+	echo Uninstall finished.
+	echo Reboot the system.
+	RED='\033[0;31m'
+	BOLD=$(tput bold)
+	echo -e "${RED} ${BOLD} If you have a 2022 or 2023 model, please help testing the new features '\e]8;;https://github.com/johnfanv2/LenovoLegionLinux/issues/46\ahere\e]8;;\a'"
+}
```
Comment (ptr1337) [inline lenovolegionlinux-dkms/lenovolegionlinux.install:18]: new line please

File: lenovolegionlinux/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,64 @@
+# Maintainer: MrDuartePT <gonegrier.duarte@gmail.com>
+# Maintainer: johnfanv2 <https://github.com/johnfanv2>
+_pkgname=lenovolegionlinux
+pkgname=${_pkgname}
+pkgver=0.0.18
+pkgrel=0
+pkgdesc="LenovoLegionLinux (LLL) brings additional drivers and tools for Lenovo Legion series laptops to Linux. PLEASE READ THE REPO BEFORE INSTALL THIS PACKAGE!!!"
+arch=("x86_64")
+url="https://github.com/johnfanv2/LenovoLegionLinux"
+license=('GPL')
```
Comment (ptr1337) [inline lenovolegionlinux/PKGBUILD:10]: SPDX Identifier

File: lenovolegionlinux/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+)
+makedepends=(
+  python-build
+  python-installer
+  python-setuptools
+  python-wheel
+  libinih
+)
+optdepends=(
+  "lenovolegionlinux-dkms-git: DKMS module"
```
Comment (ptr1337) [inline lenovolegionlinux/PKGBUILD:27]: Change to stable please

File: lenovolegionlinux/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+)
+
+conflicts=(
+  legion-fan-utils-linux-git
```
Comment (ptr1337) [inline lenovolegionlinux/PKGBUILD:31]: Change to stable please

File: lenovolegionlinux-dkms/PKGBUILD (hunk 1 above)
Comment (MrDuartePT) [reply lenovolegionlinux-dkms/PKGBUILD:21]: Yes, forget to change that

**Review by ptr1337 — CHANGES_REQUESTED** [review]

File: lenovolegionlinux/PKGBUILD (hunk 6 above)
Comment (MrDuartePT) [reply lenovolegionlinux/PKGBUILD:31]: this conflict can be removed actually

**Review by ptr1337 — APPROVED** [review]

**Pull request merged**